### ISSUE TITLE
Logo in patient experience header is mislabelled

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#3982]
## Area or Component
Patient Experience

## Quick Summary
There's a SimpleReport logo in the patient experience labelled with the string literal `process.env.REACT_APP_TITLE`.

## Accessibility Violation
The short text alternative for the complex image is not appropriate or meaningful.
https://cdc-devtools.dequecloud.com/rule-help/en/alt-text-short-text-not-meaningful

## Action Requested
Update the short text for the logo to render properly (use `process.env.REACT_APP_TITLE` as the variable name, not the literal name)

## Source Code (optional)
https://github.com/CDCgov/prime-simplereport/blob/c3e6b2dc330021e425e01657eaee613c51f9e11c/frontend/src/patientApp/PatientHeader.tsx#L38-L38

## Original Deque Issue
alt-text-short-text-not-meaningful | The short text alternative for the complex image is not appropriate or meaningful. | Alt text for logo is not useful (PROCESS-REACT-ENV) | Short text alternative is not appropriate | https://cdc-devtools.dequecloud.com/rule-help/en/alt-text-short-text-not-meaningful | serious | Yes | No | Images | ["body > div#root:nth-of-type(1) > div.App:nth-of-type(1) > div#main-wrapper:nth-of-type(1) > header.header.border-bottom.border-base-lighter:nth-of-type(1) > div.display-flex.flex-align-center.maxw-tablet.grid-container.patient-header:nth-of-type(1) > div.padding-y-1:nth-of-type(1) > div#basic-logo.margin-bottom-0:nth-of-type(1) > div.display-flex.flex-align-center:nth-of-type(1) > img.width-4:nth-of-type(1)"] | The short text alternative for the complex image is not appropriate or meaningful. | <img class="width-4" src="/app/static/media/simplereport-logomark-color.f83dd259ac76a05b9752021f20eb0b43.svg" alt="{process.env.REACT_APP_TITLE}"> | wcag2a,wcag111c | rnv7@cdc.gov | Patient Results - DOB | https://dev.simplereport.gov/app/pxp/birth-date-confirmation?plid=2d388b8a-ce92-43e9-bc35-75306ef6df08 |   | 2022-07-14T18:59:32.829Z
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
